src/LocalAuthentication/LAEnums.cs

```diff
@@ -30,13 +30,30 @@ public enum LAStatus : nint {
 		/// Authentication could not start, because passcode is not set on the device.
 		PasscodeNotSet       = -5,
 		/// Authentication could not start, because Touch ID is not available on the device.
+		[Deprecated (PlatformName.iOS, 11,0, message: "Use 'BiometryNotAvailable' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10,13, message: "Use 'BiometryNotAvailable' instead.")]
 		TouchIDNotAvailable  = -6,	    
+
 		/// Authentication could not start, because Touch ID has no enrolled fingers.
+		[Deprecated (PlatformName.iOS, 11,0, message: "Use 'BiometryNotEnrolled' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10,13, message: "Use 'BiometryNotEnrolled' instead.")]
 		TouchIDNotEnrolled   = -7,
 
+		[Deprecated (PlatformName.iOS, 11,0, message: "Use 'BiometryLockout' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10,13, message: "Use 'BiometryLockout' instead.")]
 		TouchIDLockout       = -8,
 		AppCancel            = -9,
 		InvalidContext       = -10,
+
+		[iOS (11,0), Mac (10,13)]
+		BiometryNotAvailable = TouchIDNotAvailable,
```

Comment:
Since it's the same value (and the name does not matter) it's not actually limited to iOS 11+
That might confuse people in keeping deprecated source while they do not need to.
We should add a test for such cases (please file a bug so we don't forget)